volumes/miovision/sql/create_dq.sql

```diff
@@ -0,0 +1,61 @@
+-- Make a table to store time ranges with bad/questionable/susicious data
+-- TODO:
+/*
+ALTER TABLE scannon.miovision_qc SET SCHEMA miovision_api;
+ALTER TABLE miovision_api.miovision_qc RENAME qa;
+ALTER TABLE nwessel.qc_investigation_levels SET SCHEMA miovision_api;
+ALTER TABLE nwessel.qc_problem_levels SET SCHEMA miovision_api;
+
+CREATE FUNCTION miovision_api.update_qa_last_updated_ts()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW.last_updated = now();
+    RETURN NEW;
+END;
+$$ language 'plpgsql';
+
+CREATE TRIGGER update_qa_last_updated_ts_trigger
+BEFORE UPDATE ON miovision_api.qa
+FOR EACH ROW
+EXECUTE PROCEDURE miovision_api.update_qa_last_updated_ts();
+*/
+
+CREATE TABLE miovision_api.qa_investigation_levels (
+    uid text PRIMARY KEY,
+    description text UNIQUE NOT NULL
+);
+COMMENT ON TABLE miovision_api.qa_investigation_levels
+IS 'Indicates the furthest degree to which the movision QA issue has been investigated. Is this only a suspicion? Or has the issue been fully confirmed/resolved?';
+
+CREATE TABLE miovision_api.qa_problem_levels (
+    uid text PRIMARY KEY,
+    description text UNIQUE NOT NULL
+);
+COMMENT ON TABLE miovision_api.qa_problem_levels
+IS 'What is the nature of the problem indicated for the given subset of miovision data?';
+
+CREATE TABLE miovision_api.bad_data_ranges (
+    uid serial PRIMARY KEY,
+    intersection_uid smallint REFERENCES miovision_api.intersections (intersection_uid),
+    intersection_name text,
+    classification_uid smallint REFERENCES miovision_api.classifications (classification_uid),
+    last_updated timestamp DEFAULT NOW(),
```

Comment:
remove, replacing with logging of changes via https://github.com/Toronto-Big-Data-Innovation-Team/bdit_audit-trigger

Reply:
mentioned in TODO comment as well as the main comment above